design/design-doc-stacks.md

```diff
@@ -307,18 +393,21 @@ Possible solutions for package processing are listed below, along with their pro
   * PRO: This is the most versatile option as it can be used in contexts outside of a Crossplane cluster.
   * CON: Doesn't integrate very cleanly into the Crossplane mainline scenario with the SM.
 
-**Recommendation:** It is recommended to start with the "Job initContainer and shared volume" approach that copies the package contents from the package initContainer to a shared volume and has a Crossplane container main entry point perform the processing logic over the shared volume contents, sending out the artifacts to `stdout`.
-Stack packages should only have a few KB of yaml files, so it's a significant concern to blow that image size up to many MB right now.
+One approach could be to start with the "Job initContainer and shared volume" that copies the package contents from the package initContainer to a shared volume and has a Crossplane container main entry point perform the processing logic over the shared volume contents, sending out the artifacts to `stdout`.
```

Comment:
this rewording here sort of avoids explaining what we've actually got implemented right now.  I'd recommend updating the wording to be more explicit about what is currently implemented.

Reply:
I'll revisit this.

Reply:
Have another look.  Let me know if I am still missing the distinction or subtleties between then and now ideas.